crates/typst-library/src/meta/document.rs

```diff
@@ -30,6 +32,9 @@ pub struct DocumentElem {
     /// The document's keywords.
     pub keywords: Keywords,
 
+    /// The document's creation date. Requires a positive year, month and day. If any of these aren't given, no date is written.
```

Comment:
Could you wrap comments at 80 columns here and elsewhere?

Reply:
I assume the year+month+day requirement is because of xmp-writer's API, right? @reknih will make a new release shortly that allows more fine-grained date writing, so that we can lift it.

Reply:
I have released `xmp-writer 0.2.0` which allows writing a date if you have any of the parts, so this can be addressed with an xmp-writer bump. Cross-reference #2374 for a possible implementation.